ArduPlane/commands_logic.cpp

```diff
@@ -131,6 +131,7 @@ bool Plane::start_command(const AP_Mission::Mission_Command& cmd)
         break;
 
     case MAV_CMD_DO_LAND_START:
+        mission.set_in_landing_sequence(true);
```

Comment:
What about the `NAV_LAND` and `NAV_VTOL_LAND` case?

Reply:
Why not track this inside the library instead of requiring an external setter?

Reply:
I have moved this into the library now.  I also set the flag true for NAV_LAND and NAV_VTOL_LAND.  This is now done in start_command() line 287.